eucalyptus.te

```diff
@@ -18,6 +18,7 @@ policy_module(eucalyptus, 0.1.0)
 gen_require(`
     attribute fixed_disk_raw_read;
     attribute fixed_disk_raw_write;
+    type cyphesis_port_t;
```

Comment:
This is unnecessary since you are using the interface for connecting to this port.